### PR TITLE
Ultramag add Charlotte

### PR DESCRIPTION
**Pull request by ultramag**
Anyone that's been to Charlotte - have you done any local stuff that you've enjoyed.  

:football: 
:grinning:

### CODE REVIEW

File: Charlotte [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Charlotte is Ok..
+Good place to raise a family.
+
+
+
+Attractions:
+Parks 
+Racing
```
Comment (ultramag) [inline Charlotte]: Racing is cool..

**Comment by seannittner** [comment]
Sorry to say I've never been to Charlotte.

File: Charlotte (hunk 1 above)
Comment (seannittner) [inline Charlotte]: Like Nascar racing? Go carts? Horses?

Comment (ultramag) [inline Charlotte]: Yes Nascar and Drag Racing is popular here

**Pull request merged**